### DISCUSSION TITLE
Split background color While I only have one color

### DISCUSSION THREAD

**Discussion by MrCasquette** [tailwindlabs/tailwindcss#11948] [Help]
**What version of Tailwind CSS are you using?**

 v3.3.3

**What build tool (or framework if it abstracts the build tool) are you using?**

postcss-cli 8.4.28 Next.js 13.4.19, webpack 5.88.2

**What version of Node.js are you using?**

v20.3.0

**What browser are you using?**

Chrome, edge, Arc

**What operating system are you using?**

macOS

**Reproduction URL**

A Tailwind Play link or public GitHub repo that includes a minimal reproduction of the bug. **Please do not link to your actual project**, what we need instead is a _minimal_ reproduction in a fresh project without any unnecessary code. This means it doesn't matter if your real project is private/confidential, since we want a link to a separate, isolated reproduction anyways.

[See here](https://play.tailwindcss.com/mPWKwBxGTB?file=config)

**Describe your issue**

As you can see the background is split in two color while the body has only custom one named bg-dark-50. If you put a border 1px solid everywhere, or add opacity bg-dark-50/90 or even disable margin-0 on layout.css it fixes the problem. You can even just change the color. But unfortunately that's my background color ^^
I tried with different dark colors some work other don't. For example #1A1A1A has no problem. Hope it helps. Sure it could help me. Thank you for your time. :)

<a href="https://ibb.co/ZVHjMWm"><img src="https://i.ibb.co/ZVHjMWm/Capture-d-e-cran-2023-09-03-a-18-54-38.png" alt="Capture-d-e-cran-2023-09-03-a-18-54-38" border="0"></a>

Edit: You need a screen that can display Rec 709 or almost adobe RGB 1998. For example Apple RGB or lowest are not wide enough to display full colors

**Comment by thecrypticace** [comment]
Hey! So I can reproduce this in Chrome but not Safari after setting my HDR-capable display's color profile to Adobe RGB. **But ONLY if HDR was disabled.** I tried Rec 2020 but in non-HDR mode the dark blue appears nearly black.

I can reproduce it in Play without Tailwind involved at all:
https://play.tailwindcss.com/wts7smdPzQ

Interestingly if you open the Generated CSS panel Chrome starts rendering only a single color. So, it's definitely a rendering issue within Chrome itself. I'll look around to see if I can find any existing issues and open one with a minimal repro with them if not.

Unfortunately there's nothing _we_ can do to fix / workaround this as far as I know. Sorry :(

**Reply by MrCasquette** [reply]
Thanks for your answer this is very interresting and also very strange ^^
